Sofa/Component/SolidMechanics/FEM/HyperElastic/src/sofa/component/solidmechanics/fem/hyperelastic/TetrahedronHyperelasticityFEMForceField.inl

```diff
@@ -55,12 +56,55 @@ template <class DataTypes> TetrahedronHyperelasticityFEMForceField<DataTypes>::T
     , m_edgeInfo(initData(&m_edgeInfo, "edgeInfo", "Internal edge data"))
     , l_topology(initLink("topology", "link to the topology container"))
 {
-
+    this->d_componentState.setValue(sofa::core::objectmodel::ComponentState::Loading);
 }
 
 template <class DataTypes> TetrahedronHyperelasticityFEMForceField<DataTypes>::~TetrahedronHyperelasticityFEMForceField()
+= default;
+
+template <class DataTypes>
+void TetrahedronHyperelasticityFEMForceField<DataTypes>::instantiateMaterial()
 {
+    const std::string& material = d_materialName.getValue();
```

Comment:
An OptionGroup should probably be used here instead of a std::string for d_materialName.